packages/apputils/src/dialog.tsx

```diff
@@ -3,12 +3,12 @@
 
 import { ITranslator, nullTranslator } from '@jupyterlab/translation';
 import {
-  Button,
   closeIcon,
   LabIcon,
   ReactWidget,
   Styling
 } from '@jupyterlab/ui-components';
+import { Button } from '@jupyter/react-components';
```

Comment:
To improve the look of the close button, could you update the close button at

by something like this:

```jsx
              <Button
                appearance="stealth"
                className="jp-Dialog-close-button"
                onMouseDown={handleMouseDown}
                onKeyDown={handleKeyDown}
                title={trans.__('Cancel')}
                minimal
              >
                <closeIcon.react tag={null} />
              </Button>
```